dhis-2/dhis-services/dhis-service-schema/src/main/java/org/hisp/dhis/schema/Schema.java

```diff
@@ -605,6 +610,20 @@ public void setPropertyMap( Map<String, Property> propertyMap )
         return embeddedObjectProperties;
     }
 
+    public Map<String, Property> getAnalyticObjectProperties()
```

Comment:
Rename to "getAnalyticalObjectProperties"

Reply:
fixed